### PR TITLE
SIMSBIOHUB-507: Add Caribou Herd Layer to Survey Study Area Map

### PR DESCRIPTION
**Pull request by NickPhura**
## Links to Jira Tickets

https://apps.nrs.gov.bc.ca/int/jira/browse/SIMSBIOHUB-507

## Description of Changes

Can now pick from the caribou herd layer when defining your survey study area boundary.

## Testing Notes

Selecting/removing caribou regions works, like any other region layer we already supported

### CODE REVIEW

**Review by KjartanE — DISMISSED** [review]
Tested locally, works as intended

File: app/src/components/map/wfs-utils.tsx [hunk 1]
```diff
@@ -39,9 +45,9 @@ export const layerContentHandlers: Record<string, any> = {
             key={`${feature.id}-game-management-zone-id`}>{`Game Management Zone: ${feature.properties.GAME_MANAGEMENT_ZONE_ID}`}</div>
           <div
             key={`${feature.id}-game-management-zone-name`}>{`Game Management Zone Name: ${feature.properties.GAME_MANAGEMENT_ZONE_NAME}`}</div>
-          <div key={`${feature.id}-area`}>{`Region Area: ${(feature.properties.FEATURE_AREA_SQM / 10000).toFixed(
-            0
-          )} ha`}</div>
+          <div key={`${feature.id}-area`}>
+            {`Region Area: ${(feature.properties.FEATURE_AREA_SQM / 10000).toFixed(0)} ha`}
```
Comment (curtisupshall) [inline app/src/components/map/wfs-utils.tsx]: You may want to chain `.toLocaleString()`, which formats a number like `1234567` into `1,234,567`

**Review by curtisupshall — DISMISSED** [review]
Tested locally, works great!

File: app/src/components/map/wfs-utils.tsx (hunk 1 above)
Comment (NickPhura) [reply app/src/components/map/wfs-utils.tsx]: good suggestion, added.

**Review by curtisupshall — APPROVED** [review]
Fantastic 🤩

**Review by al-rosenthal — APPROVED** [review]
🦌

**Pull request merged**